kirakiratter.css

```diff
@@ -130,7 +130,7 @@ body /* user */,
 .column-icon,
 .column-back-button,
 .column-link {
-  background: #d0e093;
+  background: #d0e093 !important;
```

Comment:
Ah, well...these structure has heavily changed at 1.4.2, due to the pin/unpin feature.